### PR TITLE
docs: add references to API documentation

### PR DESCRIPTION
**Pull request by bobheadxi**
:tickets: **Ticket(s)**: follow-up to #571 (for #556)

---

## :construction_worker: Changes

Add references to the API documentation in the relevant places.

## :flashlight: Testing Instructions

n/a

### CODE REVIEW

File: docs_src/index.html.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ started to address these problems.
 
 This site primarily documents how to set up and use Inertia - to learn more
 about the project, check out our [GitHub repository](https://github.com/ubclaunchpad/inertia)!
-A complete command reference for the Inertia CLI is also available
-[here](/cli).
+A complete command reference for the Inertia CLI is also available [here](/cli).
+If you're interesting in building on Inertia, check out the [API reference](/api).
```
Comment (yaoharry) [inline docs_src/index.html.md]: Should be interested instead of interesting?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -185,14 +193,20 @@ Make sure you have a local daemon set up for this web app to work - see the prev
 
 ### Documentation
 
-Our [usage documentation website](https://inertia.ubclaunchpad.com/) is built with [Slate](https://github.com/lord/slate).
+Our [usage documentation website](https://inertia.ubclaunchpad.com/) is built
+with [Slate](https://github.com/lord/slate).
 The builds are in `/docs`, but the documentation source is in `/docs_src`.
 
-To build and deploy the documentation locally:
+In additional to the Slate usage guide, we also have an
```
Comment (yaoharry) [inline CONTRIBUTING.md]: Should this be "we also have a*"?

File: docs_src/index.html.md (hunk 1 above)
Comment (bobheadxi) [reply docs_src/index.html.md]: Hehe oops good catch

File: docs_src/index.html.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ started to address these problems.
 This site primarily documents how to set up and use Inertia - to learn more
 about the project, check out our [GitHub repository](https://github.com/ubclaunchpad/inertia)!
 A complete command reference for the Inertia CLI is also available [here](/cli).
-If you're interesting in building on Inertia, check out the [API reference](/api).
+If you're interetsted in building on Inertia, check out the [API reference](/api).
```
Comment (yaoharry) [inline docs_src/index.html.md]: Interested* I'm sorry 😢

Comment (bobheadxi) [reply docs_src/index.html.md]: NOOOOOO

**Pull request merged**